src/LaunchDarkly/Server/Network/Eventing.hs

```diff
@@ -69,12 +74,15 @@ eventThread manager client clientContext = do
             payloadId <- liftIO $ atomicModifyIORef' rngRef (swap . random)
             let
                 encoded = encode events'
+                compressed = if compressEvents then GZip.compress encoded else encoded
```

Comment:
Should it maybe be something like `maybeCompressed`? Or alternatively more generic like `payload`.